researchapp/services/logging.py

```diff
@@ -0,0 +1,42 @@
+""" Log stuff to ElasticSearch so we can look at it later. """
+import datetime
+import json
+import requests
+
+
+ES_URL = 'https://search-s4s-logs-xsjsafiwd7vkpiucmjqmdjkp7y.us-west-2.es.amazonaws.com/' \
```

Comment:
Pyramid likes config files, so I put it there, but also added an os.environ override.